packages/memtrace/memtrace.0.2.2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Streaming client for Memprof"
+description: "Generates compact traces of a program's memory use."
+maintainer: ["opensource@janestreet.com"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/janestreet/memtrace"
+bug-reports: "https://github.com/janestreet/memtrace/issues"
+depends: [
+  "dune" {>= "2.3"}
+  "ocaml" {>= "4.11.0"}
+]
```

Comment:
```suggestion
]
conflicts: [
  "base-domains"
]
```
```
#=== ERROR while compiling memtrace.0.2.2 =====================================#
# context              2.1.2 | linux/x86_64 | ocaml-variants.4.12.0+domains | pinned(https://github.com/janestreet/memtrace/releases/download/v0.2.2/memtrace-0.2.2.tbz)
# path                 ~/.opam/4.12/.opam-switch/build/memtrace.0.2.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p memtrace -j 31 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/memtrace-36-abe214.env
# output-file          ~/.opam/log/memtrace-36-abe214.out
### output ###
# File "test/dune", line 7, characters 0-53:
# 7 | (rule
# 8 |   (alias runtest)
# 9 |   (action (run ./trace.exe)))
# (cd _build/default/test && ./trace.exe)
# Fatal error: exception Failure("Gc.memprof.start: not implemented in multicore")
# File "test/dune", line 11, characters 0-52:
# 11 | (rule
# 12 |   (alias runtest)
# 13 |   (action (run ./fork.exe)))
# (cd _build/default/test && ./fork.exe)
# Fatal error: exception Failure("Gc.memprof.start: not implemented in multicore")
# File "test/dune", line 3, characters 0-52:
# 3 | (rule
# 4 |   (alias runtest)
# 5 |   (action (run ./test.exe)))
# (cd _build/default/test && ./test.exe)
# Fatal error: exception Failure("Gc.memprof.start: not implemented in multicore")
```

Reply:
Does opam support `{with-test}` in a conflicts field? Ideally we'd be able to install memtrace on 5.0 even though the tests don't pass. (The tests that fail are to do with actually tracing the current process. The library is still useful for reading and writing trace files even when Gc.Memprof is not working)

Reply:
This won’t work sadly. See: https://github.com/ocaml/opam/issues/4048

Reply:
:(